_monitoring-your-cluster/metrics/getting-started.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+title: Metrics Framework 
+parent: Trace Analytics
+nav_order: 65
+redirect_from:
+  - /monitoring-your-cluster/metrics/
+---
+This is an experimental feature and is not recommended for use in a production environment. For updates on the progress of the feature or if you want to leave feedback, see the associated [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/10141).    
+{: .warning}
+
+
+
+The current OpenSearch stats APIs offer valuable insights into the inner workings of each node and the cluster as a whole. However, they lack certain details such as percentiles and do not provide the semantics of richer metric types like histograms. Consequently, identifying outliers becomes challenging. OpenSearch with Metrics Framework feature adds comprehensive metric support to effectively monitor the cluster.
+
+Developers can easily add new metrics using the Metrics Framework and some of them are already added to visualize the query shaping, see the associated PR [GitHub issue](https://github.com/opensearch-project/OpenSearch/issues/10724). OpenTelemetry offers decent support for metrics, and we are leveraging the existing `telemetry-otel` plugin to provide the implementation for metrics as well. OpenSearch users can easily consume these metrics through open source OpenTelemetry sinks.
```

Comment:
I think we don't need to mention other issues / pull requests here, that would be separate features

Reply:
```suggestion
Plugin and extension developers should be able to easily add new metrics using the Metrics Framework APIs. By default, the OpenSearch distribution bundles `telemetry-otel` plugin that provide the implementation for metrics instrumentation based on [OpenTelemetry](opentelemetry.io) Java SDK.
```

Reply:
@reta: I combined this feedback with my suggestion above.